src/export/export-chakra.ts

```diff
@@ -100,9 +106,9 @@ const processIcons = async (icons: Icons): Promise<ChakraIcon[]> => {
 // Process text styles to replace hardcoded font families with tokens
 // e.g. replace `Noto Sans` with `body`
 const processTextStyles = (
```

Comment:
One of the things I like about the Chakra implementation is the way it handles responsive text styles. We can do things like <p testStyle="label"> and the label style is defined to be, for example 10px on mobile, 12px on desktop etc.
I'm not sure I can glean from this whether its attempting something similar here or leaving that as something for the consumer to do. What does the output of this look like?

Reply:
Currently this only outputs custom properties, no classes or styles, etc. but that is something I want to add.